modules/subscriptions/views.php

```diff
@@ -695,7 +695,7 @@ function jetpack_do_subscription_form( $instance ) {
 	if ( empty( $instance ) || ! is_array( $instance ) ) {
 		$instance = array();
 	}
-	$instance['show_subscribers_total'] = empty( $instance['show_subscribers_total'] ) ? false : true;
+	$instance['show_subscribers_total'] = empty( $instance['show_subscribers_total'] ) || 'false' === $instance['show_subscribers_total'] ? false : true;
```

Comment:
```suggestion
	$instance['show_subscribers_total'] = ! empty( $instance['show_subscribers_total'] ) && 'false' !== $instance['show_subscribers_total'];
```

Reply:
I'm always get confused with complex-ish logical operators, so I'd vote for something more explicit: 
```suggestion
  $instance['show_subscribers_total'] = false;
if ( ! empty( $instance['show_subscribers_total'] ) && 'false' !== $instance['show_subscribers_total'] ) {
  $instance['show_subscribers_total'] = true;
}
```